README.md

```diff
@@ -130,6 +130,20 @@ com.google.cloud.logging.LoggingHandler.formatter=java.util.logging.SimpleFormat
 java.util.logging.SimpleFormatter.format=%3$s: %5$s%6$s
 ```
 
+### Enabling gzip compression
+Tomcat offers the possibility to use GZIP compression.
+ 
+To take advantage of this feature set the property `tomcat.server.connector.compression` to `on`:
+```yaml
+env_variables:
+  TOMCAT_PROPERTIES: tomcat.server.connector.compression=on
+```
+
+You can also specify a numerical value indicating the minimal amount of data (in bytes) before using compression.
```

Comment:
How? Don't you need to allow the user to set the `compressionMinSize` attribute then?

Reply:
No, setting `compression` to a numerical value is equivalent to set `compression` to `on` and `compressionMinSize` to the numerical value.

Reply:
I see.